src/io.F

```diff
@@ -947,6 +947,117 @@ subroutine printchecks(idump)
       return
       end
 c-----------------------------------------------------------------------
+      subroutine errors_output(out1,out2,out3,out4,out5,out6,
```

Comment:
There is a fair bit of overlap with the `print_errors` routine directly below it. Only one of these routines should exist.